src/entities/addLiquidity/composable-stable/addLiquidityComposableStable.ts

```diff
@@ -206,7 +205,9 @@ export class AddLiquidityComposableStable implements AddLiquidityBase {
                 break;
             }
             default:
-                throw Error('Unsupported Add Liquidity Kind');
+                throw Error(
+                    'Unsupported Add Liquidity Kind, for Init use InitPool instead of AddLiquidity',
+                );
```

Comment:
I think it's a bit odd to add the Init specific error message to the default case.
How about we keep the Init case and move the specific Init error message there?